app/src/main/java/me/ivg2/foodapp/Model/Food.java

```diff
@@ -1,4 +1,60 @@
 package me.ivg2.foodapp.Model;
 
+import java.util.Date;
+
 public class Food {
+    private String name;
+    private double quantity;
+    private Date expirationDate;
+    private String imageURL;
+
+    public Food(String name) {
```

Comment:
You should initialize the imageURL to something, even if you explicitly set it to null.  This indicates that that is intended behaviour and not an oversight.